packages/components/src/components/post-linkarea/post-linkarea.tsx

```diff
@@ -15,19 +13,25 @@ export class PostLinkarea {
   @Element() host: HTMLPostLinkareaElement;
 
   private dispatchClick({ ctrlKey, shiftKey, altKey, metaKey }: MouseEvent) {
-    const interactiveElement: InteractiveElement =
-      this.host.querySelector(`[data-link]${INTERACTIVE_ELEMENTS_SELECTOR}`) ??
-      this.host.querySelector(INTERACTIVE_ELEMENTS_SELECTOR);
+    this.host
+      .querySelector(INTERACTIVE_ELEMENTS_SELECTOR)
+      .dispatchEvent(new MouseEvent('click', { ctrlKey, shiftKey, altKey, metaKey }));
+  }
 
-    if (!interactiveElement) {
+  componentDidLoad() {
+    const interactiveElements = this.host.querySelectorAll(INTERACTIVE_ELEMENTS_SELECTOR);
```

Comment:
Depending on how the components is used its children could be added or changed after the `post-linkarea` has loaded. Can you please use a mutation observer?